docs/preview/02-getting-started.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Getting started
+---
+
+# Getting started with Arcus Messaging
+**Welcome to Arcus Messaging!** 🎉
+
+This page is dedicated to be used as a walkthrough on how to set up Arcus Messaging in a new or existing project. Arcus Messaging is an umbrella term for a set of NuGet packages that kickstart your messaging solution.
+
+**Used terms:**
+* *message handler:* a custom implementation of an Arcus Messaging-provided interface that processes a deserialized Azure Service bus message.
+* *message pump:* an Arcus Messaging-provided registered service that receives Azure Service bus messages for you, and "pump's" them through your *message handlers*.
+
+## The basics
+> 👉 Arcus Messaging is currently only supported for [Azure Service bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) solutions.
+
+Arcus Messaging helps with receiving messages from Azure Service bus queues/topic subscriptions. Instead of directly interacting with the [`ServiceBusReceiver`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver), it allows you to implement one or more *'message handler'* interfaces. Arcus Messaging will use these custom implementations and determine based on several criteria to which  *'message handler'* it should route the message.
+
+By using *'message handlers'*, you don't have to worry about routing, deserialization, or even complete/dead-letter/abandon messages.
```

Comment:
Maybe make it clear that you can take care of abandon/deadlettering yourself as well.

Reply:
Yes, exactly. Will do.

Reply:
Done in https://github.com/arcus-azure/arcus.messaging/pull/546/commits/46b6abeffdbb53d83047190f02242588577672d9